sdcm/sct_config.py

```diff
@@ -875,7 +875,9 @@ class SCTConfiguration(dict):
                 'keep' - Keep instances running and leave credentials alone
                 'keep-on-failure' - Keep instances if testrun failed
              """,
-             choices=("keep", "keep-on-failure", "destroy"))
+             choices=("keep", "keep-on-failure", "destroy")),
+        dict(name="workaround_kernel_bug_for_iotune", env="SCT_WORKAROUND_KERNEL_BUG_FOR_IOTUNE", type=bool,
```

Comment:
Is this relevant for upgrade tests only? If yes, add this to the description

Reply:
It's a general kernel bug, I only touched in upgrade_test. If other tests use gce backend, the problem also exists.

Currently we didn't touch this problem with aws backend, the kernel version in ami is different.

Reply:
please add details, which kernel version, etc. so we will be aware when to use it

Reply:
How about this one?

`help="Workaround a known kernel bug which causes iotune to fail in scylla_io_setup, only effect GCE backend"`

Reply:
> please add details, which kernel version, etc. so we will be aware when to use it

Currently we always need to use it if gce backend is used.
We can remove or disable this workaround when the kernel bug is fixed in default kernel of GCE instance.

bad kernel:
- 3.10.0-1062.1.2.el7.x86_64
- 3.10.0-1062.4.3.el7.x86_64 (latest, still not fix)

Reply:
> > please add details, which kernel version, etc. so we will be aware when to use it
> 
> Currently we always need to use it if gce backend is used.
> We can remove or disable this workaround when the kernel bug is fixed in default kernel of GCE instance.
> 
> bad kernel:
> 
> * 3.10.0-1062.1.2.el7.x86_64
> * 3.10.0-1062.4.3.el7.x86_64 (latest, still not fix)

Update above info to commitlog.